### DISCUSSION TITLE
Chia node not syncing

### DISCUSSION THREAD

**Discussion by Jaynpp** [Chia-Network/chia-blockchain#18190] [Support]
it trys to sync and then cannot sync

**Comment by BrandtH22** [comment]
Hey @Jaynpp , As a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

If you are just upgrading to 2.1+ version then it could be either your system needs to have its blockchain database updated from v1 to v2 or your system has more than 61 cores and needs another config setting to be updated.

NOTE - make sure to fully stop chia before making any changes to the needed files.

To verify if the blockchain db needs to be updated navigate to ~/.chia/mainnet/db/ and look at the .sqlite file, if it has V1 in the title then we need to update and you can proceed with the steps outlined here:
https://docs.chia.net/faq/#how-do-i-upgrade-my-version-1-blockchain-database-to-version-2

If this is not the case then I would look at your log file located at ~/.chia/mainnet/log/debug.log for a line that indicates ValueError: max_workers must be <= 61 . If this line is present then you will need to navigate to the config file at ~/.chia/mainnet/config/config.yaml and in the full_node section update the parameter reserved_cores to 10.

If neither of these are the issue let us know and we can further debug.

**Comment by Jaynpp** [comment]
Works Great now , it's syncing now , taking some time .Is it usually like
that

**Reply by BrandtH22** [reply]
Very glad to hear it is syncing up as expected now! 
I am going to close this ticket but don't hesitate to reopen or join us in discord (https://discord.gg/chia) if any questions arise.

**Comment by Jaynpp** [comment]
Thanks but I cannot access the discord is there a password reset link?

**Reply by BrandtH22** [reply]
Most certainly, here is the discord support article for resetting a password that outlines where to reset your password on desktop or mobile:
https://support.discord.com/hc/en-us/articles/218410947-I-forgot-my-Password-Where-can-I-set-a-new-one

**Comment by Jaynpp** [comment]
Thanks, it worked. but it still does not allow me to send messages on
issues

**Reply by BrandtH22** [reply]
If you are running into issues using discord then I recommend contacting the discord support team linked in that article. If you are having issues in our server then let us know your discord handle here and we will support you in discord as best we can.
